test/parallel/test-vm-module-basic.js

```diff
@@ -124,12 +124,24 @@ const util = require('util');
 // Check to throws invalid exportNames
 {
   assert.throws(() => new SyntheticModule(undefined, () => {}, {}), {
-    message: 'The "exportNames" argument must be an Array of strings.' +
-      ' Received undefined',
+    message: 'The "exportNames" argument must be an ' +
+        'Array of unique strings.' +
+        ' Received undefined',
     name: 'TypeError'
   });
 }
 
+// Check to throws duplicated exportNames
+// https://github.com/nodejs/node/issues/32806
+{
+  assert.throws(() => new SyntheticModule(['x', 'x'], () => {}, {}), {
+    message: 'The "exportNames" argument must be an ' +
+        'Array of unique strings.' +
+        ' Received an instance of Array',
```

Comment:
could the error include which name is duplicated?

Reply:
I think `ERR_INVALID_ARG_TYPE` hasn't this feature.